msteams-platform/TOC.yml

```diff
@@ -307,6 +307,8 @@
     href: apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md
   - name: Meeting apps API references
     href: apps-in-teams-meetings/API-references.md
+  - name: Build shared channel
```

Comment:
Is this a good TOC entry?
Are developers building a shared channel?
Are we using building, creating, developing, etc. verbs interchangeably?